pkg/golinters/gofumpt.go

```diff
@@ -46,7 +46,7 @@ func NewGofumpt() *goanalysis.Linter {
 				if err != nil {
 					return nil, fmt.Errorf("unable to open file %s: %w", f, err)
 				}
-				output, err := format.Source(input, "")
+				output, err := format.Source(input, format.Options{})
```

Comment:
`go get`  with `-u` will internally update dependencies if I am not wrong. Additionally, `go get` is _not_ recommended way to install golangci-lint as per https://golangci-lint.run/usage/install/

> Note: such go get installation aren't guaranteed to work. We recommend using binary installation.

Without changing version in go.mod, your PR will cause compilation error.